src/generator-vulcanjs/generators/start/index.js

```diff
@@ -0,0 +1,10 @@
+const chalk = require('chalk');
+const VulcanGenerator = require('../../lib/VulcanGenerator');
+
+module.exports = class extends VulcanGenerator {
+
+  default() {
+    this.log(chalk.green('\nStarting your app... \n'));
+    this.spawnCommandSync('meteor', ['run']);
```

Comment:
Meteor 1.5.1 

I disagree, as the whole point is to offer to users an uniform command, which "hides" the background operations .

* It executes `meteor run`? vulcan start.
* It executes `yarn start --foo bar --dport 3200 --nocomply` ? Still vulcan start
* Now it spawns a docker ? Again, vulcan start 

If you insist, we can allow vulan run|start as an alias ? :)